core/src/main/java/org/apache/carbondata/core/datastore/filesystem/CarbonFile.java

```diff
@@ -33,6 +33,9 @@
 
   CarbonFile[] listFiles();
 
+  CarbonFile[] listCarbonFilesByCountIterator(boolean recursive, String path, int fileCount)
```

Comment:
CarbonFile already has a 'path', what is the parameter 'path'? It is confusing if there is no function description in comment

Reply:
@jackylk handled